addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -4,7 +4,7 @@ local gsToolLimits = gsToolModeOP:gsub("_multi", "").."s"
 local gsSentClasMK = "gmod_"..gsToolModeOP
 local MappingFxUID = "abcdefghijklmnopqrstuvwxyz1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
-TOOL.Tab        = "Wire"
+TOOL.Tab        = gsToolModeOP:match("%w+"):gsub("^%l", string.upper)
```

Comment:
This seems unnecessarily complicated - can we just leave this at "Wire", same for the tool name below actually please

Reply:
Reverted in d1a97f191828337d3eb16f73ce41a6784df705a1